integrationTests/vm/esdt/process/esdtProcess_test.go

```diff
@@ -1214,7 +1214,7 @@ func TestScACallsScBWithExecOnDestScAPerformsAsyncCall_NoCallbackInScB(t *testin
 	)
 
 	time.Sleep(time.Second)
-	_, _ = integrationTests.WaitOperationToBeDone(t, nodes, 2, nonce, round, idxProposers)
+	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, 2, nonce, round, idxProposers)
```

Comment:
Perhaps in the past, the linter recommended to drop the return values, since they were not used. Then, also in the past, the test was extended, and these would have been needed (below). Not providing these led to a series of hidden "round does not match" errors. Due to the more strict cleanup of transactions (this PR), the test broke. Debugging was non-trivial.

Reply:
+1